examples/draw-geometry.js

```diff
@@ -0,0 +1,29 @@
+goog.require('ol.geom.LineString');
+goog.require('ol.geom.Point');
+goog.require('ol.geom.Polygon');
+goog.require('ol.render');
+goog.require('ol.style.Circle');
+goog.require('ol.style.Fill');
+goog.require('ol.style.Stroke');
+
+
+var canvas = document.getElementById('canvas');
+var pixelRatio = ol.has.DEVICE_PIXEL_RATIO;
+canvas['width'] = 100 * pixelRatio;
```

Comment:
Yes, otherwise the canvas will be scaled to css pixels. But now that you mention it, that feels strange. Let me double check.